numbat/modules/core/datetime.nbt

```diff
@@ -0,0 +1,7 @@
+use units::si
+
+fn now() -> DateTime
+fn parse_datetime(input: String) -> DateTime
```

Comment:
It might be useful to add
```
fn parse_date(date: String) -> DateTime = parse_datetime("{date}T00:00:00Z")
```
here (or similar), so we can do things like `parse_date("2024-02-06")`.

Reply:
Just a note: We can even implement `parse_time` in Numbat with hacks like:
```py
fn today() -> String = format_datetime("%Y-%m-%d", now())
fn parse_time(time: String) -> DateTime = parse_datetime("{today()}T{time}Z")

parse_time("12:30:00")
```